projects/eigen/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "http://eigen.tuxfamily.org/index.php?title=Main_Page"
+primary_contact: "eigen-core-team@lists.tuxfamily.org"
+sanitizers:
```

Comment:
Why is ubsan disabled?

Reply:
Some general advice: Usually AFL, libFuzzer+ASAN, libFuzzer+UBSAN work.
If one of them doesn't, you should explain why, because we will ask (unless we forget =).

Reply:
> BAD BUILD: /out/solver_fuzzer does not seem to be compiled with UBSan.
Check build failed.

I got the bad build error for libFuzzer+UBSAN. It doesn't tell me why it fails. I didn't write this fuzz target. Is there a way to figure the reason out locally?

Reply:
>It doesn't tell me why it fails

The log you posted seems helpful.

Use objdump and grep to find out if the binary is instrumented with ubsan

Reply:
Another idea, can you post the (entire) output of running the UBSAN build of the fuzzer.

Reply:
The output is pretty long. Should I just post everything here? Or send you the file?

Reply:
Emailing it to me works.

Reply:
Emailed.